optuna/pruners/hyperband.py

```diff
@@ -182,6 +213,10 @@ def _get_bracket_id(self, study: "optuna.study.Study", trial: FrozenTrial) -> in
         `Hyperband paper <http://www.jmlr.org/papers/volume18/16-558/16-558.pdf>`_.
         """
 
+        if len(self._pruners) == 0:
+            return 0
+
+        assert isinstance(self._n_brackets, int)
```

Comment:
Ditto.
I don't think this necessary.

ref: https://docs.python.org/3/library/math.html#math.floor

Reply:
Ditto.